components/generate_prompts/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM --platform=linux/amd64 python:3.8-slim
+
+## System dependencies
+RUN apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install git -y
+
+# install requirements
+COPY requirements.txt /
+RUN python3 -m pip install --upgrade pip
+RUN pip3 install --no-cache-dir -r requirements.txt
+
+# Install Fondant
+# This is split from other requirements to leverage caching
+ARG FONDANT_VERSION=main
+RUN pip3 install fondant[aws,azure,gcp]@git+https://github.com/ml6team/fondant@${FONDANT_VERSION}
```

Comment:
This was structured this way for reusable components. Let's just add it to the `requirements.txt` for custom components.

Reply:
This part should also be removed @PhilippeMoussalli

Reply:
Woops forgot it